Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -258,14 +262,17 @@ public class HTTPServer: Server {
 
      ### Usage Example: ###
      ````swift
-     try server.listen(on: 8080)
+     try server.listen(on: 8080, address: "localhost")
      ````
 
      - Parameter on: Port number for new connections, e.g. 8080
-    */
-    public func listen(on port: Int) throws {
+     - Parameter address: The address of the network interface to listen on. Defaults to nil, which means this server
+                          will listen on all interfaces.
+     */
+    public func listen(on port: Int, address: String?) throws {
```

Comment:
This must match the Kitura-net version: 
`public func listen(on port: Int, address: String? = nil) throws {`